### PR TITLE
[DNM] Adds temp stracktraces to debug null pipenets

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
AA request.

image

### CODE REVIEW

File: code/modules/atmospherics/machinery/atmospherics.dm [hunk 1]
```diff
@@ -167,7 +167,9 @@ Pipelines + Other Objects -> Pipe network
 /obj/machinery/atmospherics/proc/returnPipenetAir()
 	return
 
-/obj/machinery/atmospherics/proc/setPipenet()
+/obj/machinery/atmospherics/proc/setPipenet(datum/pipeline/P)
+	if(!P)
+		stack_trace("[src] attempted to join a null pipenet at Nearby: [x], [y], [z]")
```
Comment (AffectedArc07) [inline code/modules/atmospherics/machinery/atmospherics.dm:172]:
```suggestion
		stack_trace("[src] attempted to join a null pipenet at Nearby: [x], [y], [z]")
		message_admins("[src] attempted to join a null pipenet at Nearby: [x], [y], [z] | Call a headcoder or AA")
```

**Comment by AffectedArc07** [comment]
Could you verify that this doesnt happen under normal circumstances? I dont want too much noise on this.

**Comment by S34NW** [comment]
It doesn't seem to show up on my test server when removing and adding pipes, or when blowing stuff up.

**Comment by S34NW** [comment]
We've had literally no null pipenets reported during this TM period, lol.

**Pull request closed without merging**